### ISSUE TITLE
Input shaping errors

### ISSUE THREAD

**Issue by seniordngdng** [Guilouz/Creality-K1-and-K1-Max#51]
Installed rooted firmware, moonraker, and fluidd following the guide.  did all the steps on the "fix issue with input shaper" page.  whenever i try to run Input_Shaper macro it goes through the whole process but at the very end I get these errors and have to restart the printer.  I did a factory reset and reloaded everything, but it still happens.

// The SAVE_CONFIG command will update the printer config file
// with these parameters and restart the printer.
12:34:17 // Klipper state: Shutdown
12:34:17 !! {"code":"key60", "msg":"Internal error on command:M84", "values": ["M84"]}
12:34:17 !! {"code":"key60", "msg":"Internal error on command:INPUT_SHAPER", "values": ["INPUT_SHAPER"]}

I looked in printer.cfg and it has the new values for recommended shaper type and freq, and they're different for x and y, like the example in the guide.

**Comment by Remleyfpv** [comment]
Have the exact same thing: 
![image](https://github.com/Guilouz/Creality-K1-and-K1-Max/assets/103446008/1a2e1ec7-716c-473e-91dd-04a8c7580981)

**Comment by Guilouz** [comment]
I can’t reproduce this issue. Try to remove M84 command to see.

**Comment by Remleyfpv** [comment]
![image](https://github.com/Guilouz/Creality-K1-and-K1-Max/assets/103446008/1efb3dbc-e375-4c7b-8e8b-5ab29359cc76)
 I removed M84 and it still doesnt work, do you want any files from me to check anything?

**Comment by seniordngdng** [comment]
I think I figured it out... When I went through the guide and installed everything, there are parts that say to open printer.cfg and insert certain lines "at the end", which is what I did, but it actually needs to be inserted before the grey SAVE_CONFIG section. 
Everything seems to be working fine now.

**Comment by Remleyfpv** [comment]
So how do I fix this now? update via the screen?

**Comment by seniordngdng** [comment]
If you did what I described, you can either do a factory reset and install everything again or you can fix printer.cfg directly.

**Comment by Remleyfpv** [comment]
![image](https://github.com/Guilouz/Creality-K1-and-K1-Max/assets/103446008/4d429e13-aa88-45e9-8acf-06b44f614fb7)
Is it wierd that in my printer.cfg there  are a lot of "save config"s?

**Comment by seniordngdng** [comment]
That's how mine was, so I used one of the original printer.cfg's that was saved as a backup (printer-20231004-xxxxx.cfg) from before I modified anything and basically started over.   Added the new code in the proper order, renamed that file printer.cfg and deleted the current printer.cfg, save and restart.  You'll have to redo you bed level and input shaper calibrations after that but doing that fixed the errors for me.
